### ISSUE TITLE
export ES7 class and writting a comment with the word "from"

### ISSUE THREAD

**Issue by damienleroux** [royriojas/esformatter-jsx#34]
Hi,

I have an issue when formatting this code:
```javascript
export class myClass{
  execute() {
		//comment from me
  }
}
```

If I remove either "export" or "from", all is working fine.
My test comes from sublime text 3 using jsfmt:
```
{
  // autoformat on save
  "autoformat": true,

  // array of extensions for autoformat
  "extensions": ["js", "jsx", "sublime-settings"],

  // options for jsfmt
  "options": {
    "preset": "jquery",
    "indent": {
      "value": "  "
    },
    // plugins included
    "plugins": [
      "esformatter-jsx"
    ],
    "jsx": {
      "formatJSX": true, //Duh! that's the deafault
      "attrsOnSameLineAsTag": false, // move each attribute to its own line
      "maxAttrsOnTag": 3, // if lower or equal than 3 attributes, they will be kept on a single line
      "firstAttributeOnSameLine": true, // keep the first attribute in the same line as the tag
      "alignWithFirstAttribute": false, // do not align attributes with the first tag
      "spaceInJSXExpressionContainers": " ", // default to one space. Make it empty if you don't like spaces between JSXExpressionContainers
      "htmlOptions": {
        // put here the options for js-beautify.html
      }
    },
    "options-JSON": {
      "plugins": [
        "esformatter-quotes"
      ],
      "quotes": {
        "type": "double"
      }
    },
    "node-path": "node",
    "alert-errors": true,
    "ignore-selection": false
  }
}

```

Is there any way to avoid this problem?

Thank you

**Comment by royriojas** [comment]
@damienleroux  The issue should be fixed now in `v2.3.9`

**Comment by damienleroux** [comment]
Thank you very much
